mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
It is a little redundant, though I can see why it'd be convenient for users to have both.

I just tried the same code initializing times/delays at init and it does work fine with clone, so let's move those to `init` in that case. I'll keep both unless somebody thinks it'd be better to simplify and only include one attribute instead of two.